catalog/ui/catalog-ui-search/src/main/webapp/component/navigation/navigation.hbs

```diff
@@ -15,7 +15,7 @@
     <span class="fa fa-times"></span>
     <span>Cancel Drawing</span>
 </button>
-<div class="navigation-left is-button">
```

Comment:
Moved `is-button` to the menu button so the logo wouldn't get highlighted on hover.